### DISCUSSION TITLE
Can not control my encoean devices via HA

### DISCUSSION THREAD

**Discussion by onedread23** [mak-gitdev/HA_enoceanmqtt#96] [Q&A]
Hi

i was able to add my enocean devices to HA but i can not control it via HA.

If i turn on my powerplug directly on the plug i do see the status change in HA but not from HA to the device.

What mistake did i make?

[Powerplug/OfficeDesk]
address         = 0x01928EFE
rorg            = 0xD2
func            = 0x01
type            = 0x09

Its a Permundo PSC234, it was working in FHEM in both ways.

thx in advance.

**Comment by mak-gitdev** [comment]
@onedread23 

https://github.com/mak-gitdev/HA_enoceanmqtt/discussions/79#discussioncomment-7586618
